python/nanoarrow/_lib.pyx

```diff
@@ -730,9 +761,13 @@ cdef class ArrayViewBuffers:
     cdef ArrayView _array_view
     cdef int64_t _length
 
-    def __init__(self, ArrayView array_view):
+    def __cinit__(self, ArrayView array_view):
         self._array_view = array_view
-        self._length = array_view._array._ptr.n_buffers
+        self._length = 3
```

Comment:
Just out of curiosity, the `n_buffers` member of ArrowArray is not always accurate (or not what we want here)?

Reply:
Thanks for the explanation! And yes, seeing this code here, adding `n_buffers` to the view object might be useful.